app/app.py

```diff
@@ -29,19 +30,21 @@ def configure_app(flask_app: Flask, db_url: str) -> None:
     flask_app.config['MAX_CONTENT_LENGTH'] = int(os.getenv("FLASK_MAX_CONTENT_LENGTH", "1073741824"))
 
 
-def initialize_app(flask_app: Flask) -> None:
+def initialize_app(flask_app: Flask, config=config) -> None:
     from models import db, db_url
     from apis import api
-    from auth import auth
-    from utils.env_loader import DIR_KUBE_CONFIG, config
+    from auth import Auth, auth
 
     if not os.path.isdir(DIR_KUBE_CONFIG):
         os.makedirs(DIR_KUBE_CONFIG)
     configure_app(flask_app, db_url())
 
     api.init_app(flask_app)
     if 'auth' in config:
-        auth.init_app(flask_app, api)
+        Auth.enabled(True)
+        auth.init_app(flask_app, api, config['auth'])
+    else:
+        Auth.enabled(False)
```

Comment:
We don't need to set `False` value if we set `False` as a default value.

Reply:
Since this value is a static variable, when executing multiple tests, it is necessary to explicitly specify in each TestCase.

Reply:
I see.
BTW, can't we use `auth.is_enabled()` instead? Since we initialize `auth=Auth()`, we can use it likewise `db` of model instance.

Reply:
fixed.
https://github.com/rekcurd/drucker-dashboard/pull/49/commits/6adc860c79a50e14832223834d2d9d222f883e02